packages/api-db-mongodb/src/migration.ts

```diff
@@ -17,7 +18,7 @@ export const Migrations: Migration[] = [
         strict: true
       })
 
-      await migrations.createIndex({name: 1}, {unique: true})
+      await migrations.createIndex({version: 1}, {unique: true})
```

Comment:
@per2plex i guess this needs to be fixed manually in bajour. Otherwise migrations will break.